Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3DResample.h

```diff
@@ -93,7 +93,7 @@ class DiffusionTensor3DResample
 protected:
   DiffusionTensor3DResample();
 
-  void ThreadedGenerateData( const OutputImageRegionType & outputRegionForThread, ThreadIdType threadId ) ITK_OVERRIDE;
+  void DynamicThreadedGenerateData( const OutputImageRegionType & outputRegionForThread) override;
```

Comment:
In the commit message, the conditional time was used `we could just change the name of the function: ThreadedGenerateData to: DynamicThreadedGenerateData and delete the unused threadId argument.`

Does it means that this change is not needed to fix the actual bug ?

Reply:
No, it's a grammatical error, it should be can instead of could. I applied that statement to a few cases in this commit.

Reply:
Thanks for clarifying.

Then, is it correct to say that (1) changing from  `ThreadedGenerateData` to `DynamicThreadedGenerateData` and (2) calling `DynamicMultiThreadingOff` are both required to fix the failing tests ?

Reply:
That's correct. They are the two possible strategies, depending on the case (I mean, they cannot be applied together to the same case). And both were applied in this commit.